.circleci/config.yml

```diff
@@ -1031,6 +1031,7 @@ jobs:
     steps:
       - checkout
       - skip-if-docs-only
+      - skip-if-webui-only
```

Comment:
**super nit:** `skip-if-web-only` instead of `skip-if-webui-only`? or maybe @trentwatt 's PR already covers this

Reply:
hmm, IMO it's best to wait for the pr that's updating these to do it. as is most of the references are `webui` 

(21 occurrences here alone)